lib/internal/bootstrap_node.js

```diff
@@ -504,7 +513,7 @@
   };
 
   NativeModule.wrapper = [
-    '(function (exports, require, module, __filename, __dirname) { ',
+    '(function (exports, require, module, __filename, process) { ',
```

Comment:
> Its confusing that there are two wrappers now, and they are different.

That's the price of progress.

> Am I reading this correctly, internal modules no longer have __dirname?

They never really did, it was always undefined because built-in modules don't live on disk.

> I guess it would be too big a change to have `process` be injected as a module argument for all modules, internal and external, rather than be in `globals`?

I'm not against that per se but that's a bigger change and probably needs more discussion.

> why did `require()` become a local?

I believe it always was.  It sort of has to be because it works relative to the calling script.